src/main/java/bio/terra/cli/command/resource/update/GcsObject.java

```diff
@@ -54,7 +54,8 @@ protected void execute() {
             .resourceFields(updateResourceParams)
             .bucketName(newBucketName.getNewBucketName())
             .objectName(newObjectName)
-            .originalResource(resource)
+            .originalObjectName(resource.getObjectName())
```

Comment:
nit: Add a `TODO (PF-...)` comment here about removing the original value fields in the future. Ditto for the other update command classes.